papaparse.js

```diff
@@ -1457,6 +1457,34 @@ License: MIT
 			if (!input)
 				return returnable();
 
+			// Rename headers if there are duplicates
+			if (config.header)
+			{
+				let firstLine = input.split(newline)[0]
+				let separator = '_';
+				let headers = firstLine.split(delim) 
```

Comment:
Could you add `headers`variable after `firstLine`? As they are linked it make sense to define one after the other.